app/views/wands/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="container">
+  <ul>
+    <% @wands.each do |wand| %>
+    <li><%= wand.capitalize %></li>
```

Comment:
You can't just do wand.capitalize, because it is passing us a hash remember. So you have to do wand.name.capitalize